content/source/docs/enterprise/install/installer.html.md

```diff
@@ -192,6 +192,7 @@ If your instance can access the Internet, use the Online install mode.
 1. From a shell on your instance:
     * To execute the installer directly, run `curl https://install.terraform.io/ptfe/stable | sudo bash`
     * To inspect the script locally before running, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh`.
+    * To install on RedHat Enterprise Linux where pre-installation of Docker is necessary, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh no-docker`. 
```

Comment:
This suggestion adds a sub-bullet detailing the RHEL specific instructions.

```suggestion
      * RedHat Enterprise Linux requires Docker to be pre-installed. As such, execute the installer script using `sudo bash install.sh no-docker` to prevent the installer script from automatically installing Docker.
```